make/photon/prepare/templates/trivy-adapter/env.jinja

```diff
@@ -8,3 +8,6 @@ SCANNER_TRIVY_REPORTS_DIR=/home/scanner/.cache/reports
 SCANNER_TRIVY_VULN_TYPE=os,library
 SCANNER_TRIVY_SEVERITY=UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
 SCANNER_TRIVY_IGNORE_UNFIXED=false
+HTTP_PROXY={{trivy_http_proxy}}
```

Comment:
The trivy_http_proxy is a template variable, you should define it before using.
see: https://github.com/goharbor/harbor/blob/master/make/photon/prepare/utils/configs.py
https://github.com/goharbor/harbor/blob/master/make/photon/prepare/main.py
https://github.com/goharbor/harbor/blob/master/make/photon/prepare/utils/trivy_adapter.py

Reply:
@stonezdj When I add `trivy` to the list of proxy components:

```yaml
proxy:
  http_proxy: http://someproxy
  https_proxy: https://someproxys
  no_proxy:
  components:
    - core
    - jobservice
    - clair
    - trivy
```

the `trivy_http_proxy` variable is initialized dynamically:
 
```
SCANNER_LOG_LEVEL=info
SCANNER_STORE_REDIS_URL=redis://redis:6379/5
SCANNER_STORE_REDIS_NAMESPACE=harbor.scanner.trivy:store
SCANNER_JOB_QUEUE_REDIS_URL=redis://redis:6379/5
SCANNER_JOB_QUEUE_REDIS_NAMESPACE=harbor.scanner.trivy:job-queue
SCANNER_TRIVY_CACHE_DIR=/home/scanner/.cache/trivy
SCANNER_TRIVY_REPORTS_DIR=/home/scanner/.cache/reports
SCANNER_TRIVY_VULN_TYPE=os,library
SCANNER_TRIVY_SEVERITY=UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
SCANNER_TRIVY_IGNORE_UNFIXED=false
HTTP_PROXY=http://someproxy
HTTPS_PROXY=https://someproxys
NO_PROXY=chartmuseum,core,postgresql,jobservice,notary-signer,redis,127.0.0.1,db,.local,.internal,registry,portal,clair,registryctl,notary-server,log,localhost,clair-adapter,nginx,trivy-adapter
```

See https://github.com/goharbor/harbor/blob/master/make/photon/prepare/utils/configs.py#L231